lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -54,25 +55,58 @@ function formatNumber(number) {
   return number.toLocaleString(undefined, {maximumFractionDigits: 1});
 }
 
+/**
+ * Format time.
+ * @param {string} date
+ * @return {string}
+ */
+function formatDateTime(date) {
+  const options = {
+    month: 'numeric', day: 'numeric', year: 'numeric',
+    hour: 'numeric', minute: 'numeric', timeZoneName: 'short'
+  };
+  let formatter = new Intl.DateTimeFormat('en-US', options);
```

Comment:
I believe there's an issue outstanding for our permanent use of `en-US`, since we're starting fresh, thoughts on just using `.toISOString()` for now and not bringing our baggage with us yet? :)